tests/test_codesyncd/buffer_handler.test.js

```diff
@@ -159,6 +164,68 @@ describe("bufferHandler", () => {
         return true;
     };
 
+    const addTab = (repoPath, created_at, tabs ) => {
+        console.log(`repoPath: ${repoPath}, created_at: ${created_at}, tabs: ${tabs}`);
+        const tab_handler = new tabEventHandler(repoPath)
+        const config_utils = new ConfigUtils();
+        const repoId = config_utils.getRepoIdByPath(repoPath);
+        // Add tabs to buffer
+        tab_handler.addToBuffer(repoId, created_at, tabs);
+        const pathData = fs.readdirSync(tabsRepo);
+        console.log(`pathData: ${JSON.stringify(pathData)}`);
+        for (const ymlFile of pathData) {
+            console.log(`ymlFile: ${ymlFile}`);
+            const tabData = readYML(path.join(tabsRepo, ymlFile));
+            console.log(`tabData: ${JSON.stringify(tabData)}`);
+
+        }
+    }
+    const tabPath1 = "file1.txt"
+    const tabPath2 = "file2.txt"
+    const returnTabs = () => {
+        return ([
```

Comment:
@saadberry Server expects `is_active_tab` instead of `is_active`, why is there difference?

![image](https://github.com/codesyncapp/codesync-vs/assets/9863016/cd3af13f-15e1-4b49-b857-13d2acd9251f)

Reply:
Oh yes, my oversight. Fixed